app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="jumbotron text-center">
+    <h1 class="display-4">Notation</h1>
+    <p class="lead">Why wait? Note down your thoughts, to-dos .... at any time, any place with style</p>
+    <% if current_user %>
+      <a class="btn btn-primary btn-lg" href="/users" role="button">Try Now!</a>
```

Comment:
question: Why would we show this page if the user is already logged in? Wouldn't we show the notes view instead?

Reply:
That's right. Since I haven't created the notes views I redirected it to this page.